pkgs/development/ocaml-modules/hxd/default.nix

```diff
@@ -0,0 +1,39 @@
+{ lib, buildDunePackage, fetchurl
+, dune-configurator, cmdliner, angstrom
+, rresult, stdlib-shims, fmt, fpath
+}:
+
+buildDunePackage rec {
+  pname = "hxd";
+  version = "0.2.0";
+
+  useDune2 = true;
+
+  src = fetchurl {
+    url = "https://github.com/dinosaure/hxd/releases/download/v${version}/hxd-v${version}.tbz";
+    sha256 = "1lyfrq058cc9x0c0hzsf3hv3ys0h8mxkwin9lldidlnj10izqf1l";
+  };
+
+  nativeBuildInputs = [
+    dune-configurator
+  ];
+
+  buildInputs = [
+    cmdliner
+  ];
+
+  propagatedBuildInputs = [
+    angstrom
+    stdlib-shims
+    rresult
+    fmt
+    fpath
+  ];
```

Comment:
Many of these libraries need not be propagated. `stdlib-shims` only seems enough.

Reply:
Seems like it!